programs/keeper/Keeper.cpp

```diff
@@ -355,10 +355,7 @@ try
 
     std::string include_from_path = config().getString("include_from", "/etc/metrika.xml");
 
-    if (config().has(DB::PlacementInfo::PLACEMENT_CONFIG_PREFIX))
-    {
-        PlacementInfo::PlacementInfo::instance().initialize(config());
-    }
+    PlacementInfo::PlacementInfo::instance().initialize(config());
```

Comment:
Every Keeper instance will fail to start without a placement config now? Is this desired?

Reply:
> Every Keeper instance will fail to start without a placement config now?

Why?

Reply:
Nvm, I for some reason I thought `initialize` will throw `LOGICAL_ERROR` if its config is not present.